src/native-common/Image.tsx

```diff
@@ -101,20 +101,23 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         }
 
         const additionalProps = this._getAdditionalProps();
+        const extendedProps: RN.ExtendedImageProps = {
+            source: imageSource,
+            shouldRasterizeIOS: this.props.shouldRasterizeIOS,
```

Comment:
This `prop` related to [`RN.View`](https://github.com/facebook/react-native/blob/master/Libraries/Components/View/ViewPropTypes.js#L101), and there is no in [`RN.Image`](https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageProps.js#L24)

Reply:
Good point. This is a long-standing bug in reactxp. I'll remove it from the RX.Image interface and the associated documentation since it clearly doesn't do anything in RN iOS.